### ISSUE TITLE
Failing to retrieve results for genes that produce results months back and from the example genes

### ISSUE THREAD

**Issue by MikeKatz45** [cbg-ethz/rDGIdb#7]
Hello, I've recently had the need to re-run an analysis I did a few months ago and was surprised to see I couldn't reproduce the results. Upon trying with a minimal example (vignette example) I realized that the package is currently not able to retrieve any info from the database using the R wrapper. A co-worker also had this problem working in a different machine. I opened a bioconductor thread a few days ago with more details [(post)](https://support.bioconductor.org/p/p133022/) but no one seemed to know what was going on.

**Comment by LBosshard** [comment]
Hello, can you install the latest version (1.16) from [Bioconductor](https://bioconductor.org/packages/release/bioc/html/rDGIdb.html) 
and retry:
`library(rDGIdb)`
`genes <- c("TNF", "AP1", "AP2", "XYZA")`
`result <- queryDGIdb(genes)`
`resultSummary(result)`
Please let me know if the problem persists.

**Comment by MikeKatz45** [comment]
I uninstalled the package using `remove.packages()` and re-installed using the link provided. The problem persists, here is the `sessionInfo()` output.

`R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19041)

Matrix products: default

locale:
[1] LC_COLLATE=Spanish_Mexico.1252  LC_CTYPE=Spanish_Mexico.1252   
[3] LC_MONETARY=Spanish_Mexico.1252 LC_NUMERIC=C                   
[5] LC_TIME=Spanish_Mexico.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] rDGIdb_1.14.0

loaded via a namespace (and not attached):
[1] httr_1.4.2          BiocManager_1.30.10 compiler_4.0.3     
[4] R6_2.4.1            tools_4.0.3         curl_4.3           
[7] jsonlite_1.6.1 `

The problem seems to be that `BiocManager` does not install 1.16 by default. It could also be a problem on my side, but I can't think of what. I also tried uninstalling and re-installing `BiocManager` and then re-installing `rDGIdb`, but version 1.14 keeps getting installed instead of 1.16.

**Comment by LBosshard** [comment]
Do you get an error message for the installation when you try:
`remove.packages("rDGIdb")`
`.rs.restartR()`
`BiocManager::install("rDGIdb",version = 3.12)`

**Comment by MikeKatz45** [comment]
Yeah, I did get an error message. However this was the way to go to solve it as after the error message, another message said that I should do `BiocManager::install(version = 3.12)`. Here is what I did:

```
remove.packages("rDGIdb")
.rs.restartR() # or manually restart R
 BiocManager::install(version = 3.12) # said yes to all prompts
BiocManager::install("rDGIdb",version = 3.12)

library(rDGIdb)
genes <- c("TNF", "AP1", "AP2", "XYZA")
result <- queryDGIdb(genes)
resultSummary(result)
```
Everything is working fine now, what happened here? Just an outdated BiocManager? It is a bit weird since I tried uninstalling and installing BiocManager before. I'd expect this to update BiocManager when re-installing but that did not happen.

**Comment by LBosshard** [comment]
I am glad that rDGIdb is working now.
It is indeed weird that the update was not done automatically when re-installing. Maybe the package was still loaded in the r session. It cannot be updated while the package is loaded.

**Comment by wfma** [comment]
hey all i also had the same problem just now- updating biocmanager and then rdgidb seems to fix it!
